src/test/java/io/beanmapper/BeanMapperTest.java

```diff
@@ -1269,14 +1272,28 @@ void strictMultipleBeanMismatches() {
                 .addBeanPairWithStrictSource(SourceCStrict.class, TargetCNonStrict.class)
                 .addBeanPairWithStrictSource(SourceDStrict.class, TargetDNonStrict.class);
         var exception = assertThrows(BeanStrictMappingRequirementsException.class, builder::build);
-        assertEquals(SourceAStrict.class, exception.getValidationMessages().get(0).getSourceClass());
-        assertEquals("noMatch", exception.getValidationMessages().get(0).getFields().get(0).getName());
-        assertEquals(TargetBStrict.class, exception.getValidationMessages().get(1).getTargetClass());
-        assertEquals("noMatch", exception.getValidationMessages().get(1).getFields().get(0).getName());
-        assertEquals(SourceCStrict.class, exception.getValidationMessages().get(2).getSourceClass());
-        assertEquals("noMatch1", exception.getValidationMessages().get(2).getFields().get(0).getName());
-        assertEquals("noMatch2", exception.getValidationMessages().get(2).getFields().get(1).getName());
-        assertEquals("noMatch3", exception.getValidationMessages().get(2).getFields().get(2).getName());
+
```

Comment:
While a little more verbose, I actually really like the readability this change offers.